code/src/index.css

```diff
@@ -1,13 +1,174 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul,
+ol,
+li,
+p,
+pre,
+blockquote,
+figure,
+hr {
+  margin: 0;
+  padding: 0;
+}
+
+body,
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  background-color: black;
+  color: white;
+  font-family: Helvetica, Sans-Serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h3 {
+  display: block;
+  padding: 15px 0 5px 0;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+}
+
+.icon-more {
+  width: 20px;
+}
+
+.icon-more circle {
+  fill: #fff;
+}
+
+.icon-play {
+  width: 80px;
+  margin: 0 10px;
+  transition: all 100ms ease-out;
+}
+
+.icon-play:hover {
+  transform: scale(1.1);
+}
+
+.icon-play path {
+  fill: white;
+  fill-rule: nonzero;
+}
+
+.icon-heart {
+  width: 30px;
+}
+
+.icon-heart path {
+  fill: #fff;
+}
+
+.album {
+  display: flex;
+  width: 302px;
+  margin: 0 auto;
+  flex-direction: column;
+  align-content: flex-end;
+}
+
+.album-list {
+  display: grid;
+  grid-gap: 30px;
+  margin: 20px 0 20px 0;
+  grid-template-columns: repeat(1, 304px);
```

Comment:
Nice with css grid! I think this would've been nicer with a fluid width (I see you reflected on that in your comments, too). Maybe with a max width (`minmax` in css grid land)

Reply:
I will definitely look into the minmax option in my second attempt. All in all, it was a great project to work with and I learned a lot. Now I know what to change in my second version and what to do differently in upcoming projects.

Thanks for great feedback!